dlpy/applications/darknet.py

```diff
@@ -69,17 +70,27 @@ def Darknet_Reference(conn, model_table='Darknet_Reference', n_classes=1000, act
         that are larger than those sizes are cropped.
         Valid Values: 'none', 'unique', 'randomresized', 'resizethencrop'
         Default: 'unique'
+    random_mutation : string, optional
+        Specifies how to apply data augmentations/mutations to the data in the input layer.
+        Valid Values: 'none', 'random'
+        Default: 'NONE'
 
     Returns
     -------
     :class:`Sequential`
 
     '''
 
+    conn.retrieve('loadactionset', _messagelevel='error', actionset='deeplearn')
+
+    # get all the parms passed in
+    parameters = locals()
+
     model = Sequential(conn=conn, model_table=model_table)
 
-    model.add(InputLayer(n_channels=n_channels, width=width, height=height, scale=scale,
-                         random_flip=random_flip, random_crop=random_crop))
+    # get the input parameters
+    input_parameters = get_layer_options(input_layer_options, parameters)
```

Comment:
this is just to eliminate parameters that has none, still not convinced that we need this